cms/static/js/views/modals/course_outline_modals.js

```diff
@@ -859,7 +934,7 @@ define(['jquery', 'backbone', 'underscore', 'gettext', 'js/views/baseview',
                     tabs[1].editors = [StaffLockEditor];
                 } else if (xblockInfo.isSequential()) {
                     tabs[0].editors = [ReleaseDateEditor, GradingEditor, DueDateEditor];
-                    tabs[1].editors = [ContentVisibilityEditor];
+                    tabs[1].editors = [ContentVisibilityEditor, ShowCorrectnessEditor];
```

Comment:
@cahrens d51d23ee61 creates a new Visibility tab, and shows the Advanced tab only if special exams and/or prerequisite settings are enabled.  9d38fba8d0 fixes the acceptance tests broken by that change.  I'm provisioning a new sandbox server now, but have updated the screenshots on the PR so you can see what it looks like.

@roderickmorales For the label font size, I used the existing [`%t-title7`](https://github.com/edx/edx-platform/blob/6d2468d0dcf782a7561ecc61946146d6fae2762f/lms/static/sass/elements/_typography.scss#L67-L71), which uses `font-size: 14px;`. There is no existing title size that uses 15px; they go from 12px, 14px, 16px, ...  For the message text underneath, I used [`%t-copy-sub2`](https://github.com/edx/edx-platform/blob/6d2468d0dcf782a7561ecc61946146d6fae2762f/lms/static/sass/elements/_typography.scss#L116-L120), which uses `font-size: 12px;`.  Please see the screenshots in the PR description for the updated dialog.  Is that OK?